samples/boards/bbc_microbit/line_follower_robot/README.rst

```diff
@@ -0,0 +1,55 @@
+Line follower robot using micro:bit board and DFRobot Maqueen chassis
```

Comment:
Please take a look at the documentation for other samples to see how the doc is formatted and [written using restructuredText](https://docs.zephyrproject.org/latest/guides/documentation/index.html).